core/domain/learner_progress_services.py

```diff
@@ -86,9 +86,9 @@ def get_last_playthrough_information(last_playthrough_model):
         last_playthrough_model.last_played_state_name)
 
 
-def save_completed_activities(activities_completed):
+def _save_completed_activities(activities_completed):
     """Save an activities completed domain object as an
-    CompletedActivitiesModel entity in the datastore.
+    CompletedActivities model in the datastore.
```

Comment:
Generally the correct phrasing here is "a CompletedActivitiesModel instance". Ditto below.